### PR TITLE
The gibber produces properly named monkey meat.

### PR DESCRIPTION
**Pull request by harmonyn**
Fixes https://github.com/tgstation/-tg-station/issues/17572. (The bug didn't apply to regular butchering, just the gibber.)

fucking gibber code

### CODE REVIEW

File: code/modules/food&drinks/kitchen machinery/gibber.dm [hunk 1]
```diff
@@ -204,7 +204,7 @@
 	for (var/i=1 to meat_produced)
 		var/obj/item/weapon/reagent_containers/food/snacks/meat/slab/newmeat = new typeofmeat
 		var/obj/item/stack/sheet/animalhide/newskin = new typeofskin
-		newmeat.name = sourcename + newmeat.name
+		newmeat.name = "[sourcename] meat"
```
Comment (MMMiracles) [inline code/modules/food&drinks/kitchen machinery/gibber.dm]: this is gonna mess up any slab that isn't directly called 'meat' (skeleton and zombie slabs to be specific)

Comment (Supermichael777) [inline code/modules/food&drinks/kitchen machinery/gibber.dm]: "[sourcename] [name]" would work right?

Comment (phil235) [inline code/modules/food&drinks/kitchen machinery/gibber.dm]: "[sourcename] [newmeat.name]"

Comment (Supermichael777) [inline code/modules/food&drinks/kitchen machinery/gibber.dm]: Right

**Pull request merged**